src/common/transformers/DEFAULT.js

```diff
@@ -0,0 +1,32 @@
+export const defval = (value, alternative) => {
+    if (!value) {
+        return alternative;
+    }
+    return value;
+};
+
+const transformation = (_, args) => (value) => {
+    const alternative = args.find(a => a.name === 'alternative');
+
+    if (!alternative) {
+        throw new Error('Invalid Argument for DEFAULT transformation');
+    }
+
+    return new Promise((resolve, reject) => {
+        try {
+            resolve(defval(value, alternative.value));
+        } catch (error) {
+            reject(error);
+        }
+    });
+};
+
+transformation.getMetas = () => ({
+    name: 'DEFAULT',
+    type: 'transform',
+    args: [
+        { name: 'alternative', type: 'string' },
```

Comment:
At the import, all fields are "string" so if we want "number", we can add a CAST transformer. And when we can chain DEFAULT and NUMBER to generate a default "number" value